src/components/pages/InntektStep.tsx

```diff
@@ -99,7 +100,7 @@ const InntektStep = () => {
                   ? state.gradertUttak.grad
                   : ''
             }
-            style={{ width: '5rem' }}
+            className="singleSelect"
```

Comment:
I og med at du kalte de to andre for `selectAar` og `selectMaaneder`, ville jeg nok kalle denne for `selectGrad`. 
Hvis du heller ønsker å gjøre dem mer generiske ville jeg da gå for et annet mønster som fungerer for alle.